.gitignore

```diff
@@ -3,3 +3,5 @@ bin
 composer.phar
 composer.lock
 phpunit.xml
+.idea/
```

Comment:
this has to be part of your global ignore file. http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

Reply:
You're right.